### ISSUE TITLE
Lost recurrence setting after syncing with Todoist

### ISSUE THREAD

**Issue by vennetong** [haydenull/logseq-plugin-agenda#214]
### Search first

- [X] I searched and no similar issues were found

### What Happened?

Hi There,

I'm using Agenda's todoist plugin to sync with Todoist, it works fine in the most cases.

But when I tried to sync a recurrent task in Todoist, after syncing, the original task's recurrence setting will be removed and updated to a normal task. It means once I completed it in LogSeq, it also updates its status in Todoist, but this task won't be repeated itself anymore.

And more, the agenda sync cannot load the subtasks, I'm not sure it is a limit from Todoist or just a complex feature that Agenda doesn't support yet.

### Reproduce the Bug

1. create a recurrent task in Todoist 
2. sync task from Todoist
3.  update the task
4. check back the status of this task in Todoist

### Environment Information

macOS

### Are you willing to submit a PR? If you know how to fix the bug.

- [ ] I'm willing to submit a PR (Thank you!)

**Comment by haydenull** [comment]
I'm sorry, the plugin currently doesn't support recurring tasks.

**Comment by vennetong** [comment]
Dear Hayden,

No need to say sorry. I should appreciate to your efforts on this amazing plugin in Logseq communities.

I built a task system like Org-mode and generated recurring tasks ( I used to use Emacs as well), the problem is daily recurring tasks will record each state change in log and will exceed the block limitation by default. I don't want to enlarge the limitation due to performance reasons, that's why I looked forward to seeing if Todoist recurring&nbsp;&nbsp; management can be synced by Agenda.

So far everything is good, the limitation is not the key feature and I don't need to keep every log. So that's fine to me to keep the current status.

I have to say thanks again to you, the agenda plugin is the best and must for me in LogSeq

Hope everything goes fine with you there!

Venne Tong

Experienced Human Resource Consultant with pure technical gene inside, runner for the marathon race and prefer to the health life style always
